2-mandatory/2-financial-times.js

```diff
@@ -4,34 +4,71 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
-function potentialHeadlines(allArticleTitles) {
-    // TODO
```

Comment:
This look almost perfect!
One small point - keep an eye on your indentation (spacing). It sounds like a small thing, but it can make your code much more readable for other developers 😄